client/src/components/VideoCall2.jsx

```diff
@@ -0,0 +1,41 @@
+import { JitsiMeeting } from '@jitsi/react-sdk';
```

Comment:
Why do we need a second VideoCall component? I'm worried that this will mean we have two different pieces to maintain that share a lot of similar code. If there are differences between this one and the other, we should find a way to make them parameters within the same component.